schema.sql

```diff
@@ -1,5 +1,16 @@
-/* Database schema to keep the structure of entire database. */
-
+-- Database schema to keep the structure of entire database.
+/* Create a table named animals with the following columns:
+ id: integer
+ name: string
+ date_of_birth: date
+ escape_attempts: integer
+ neutered: boolean
+ weight_kg: decimal */
```

Comment:
- [ ] @dicodiaz I would recommend you remove this commented code, it's not best practice to have the commented code in your codebase.